meta/main.yml

```diff
@@ -22,6 +22,7 @@ galaxy_info:
       versions:
         - 6
         - 7
+        - 8
```

Comment:
Thanks!
Can you please also add similar meta information for other affected roles?

Check `roles/StackStorm.*/meta/main.yml` files.